torch/testing/_internal/common_methods_invocations.py

```diff
@@ -635,6 +635,35 @@ def sample_inputs_gather(op_info, device, dtype, requires_grad):
                         0, torch.tensor(0, dtype=torch.int64, device=device))),
             )
 
+def sample_inputs_take_along_dim(op_info, device, dtype, requires_grad):
+    return (SampleInput((make_tensor((S, S), device, dtype,
```

Comment:
These are going to need a tweak since SampleInput's "input" attribute must be a tensor now. So the pattern is:

```
SampleInput(make_tensor(...), args=(...))
```